components/Bookmarks/queries.js

```diff
@@ -30,6 +30,7 @@ export const getCollectionItems = gql`
         after: $cursor
         progress: $progress
         uniqueDocuments: true
+        lastDays: 30
```

Comment:
should be a null-able variables and only set to 30 when in «weiterlesen» view

Reply:
ah ja logo XD